dib/src/pages/MainPage.js

```diff
@@ -1,19 +1,60 @@
-import NavBar from '../components/NavBar'
-import './MainPage.css'
-import SearchBar from '../components/SearchBar'
-import RestList from "../components/RestList"
+import NavBar from '../components/NavBar';
+import SearchBar from '../components/SearchBar';
+import RestList from '../components/RestList';
+import Filter from '../components/Filter';
+import Button from '../components/Button';
+import { IoStar } from 'react-icons/io5';
+import { GoSearch } from 'react-icons/go';
+import './MainPage.css';
+
+function MainPage() {
+
+  const handleClick = () => {};
+
+  const handlePriceSelect = (selectedPrice) => {
+    console.log("Selected price:", selectedPrice);
+  };
+
+  const handleRatingSelect = (selectedRating) => {
+    console.log("Selected rating:", selectedRating);
+  };
+
+  const renderStars = (num) => {
+    let stars = [];
+    for (let i = 0; i < num; i++) {
+      stars.push(<IoStar key={i} />);
+    }
+    return <div className="star-wrapper">{stars}</div>;
+  };
 
-function MainPage () {
-  console.log('main page before jsx')
   return (
     <div>
-      <NavBar/>
+      <NavBar />
       <h1 className="project-name">Drop into Berlin</h1>
-     <SearchBar/>
-     <h2 className='rest-list-header'>Some great Restaurants from around the city</h2>
-     <RestList/>
+      <div className="search-container">
+        <SearchBar />
```

Comment:
In the other Components where i added Icons I used the Icons5 libary.
If its no problem for you i would say let's sta consistent in one libary.
https://react-icons.github.io/react-icons/icons?name=io5